terraform/040-id-broker/task-definition-cron.json

```diff
@@ -365,11 +369,18 @@
     "readonlyRootFilesystem": null,
     "image": "${docker_image}",
     "command": [
-      "/data/run-cron.sh"
+      "/data/run.sh"
```

Comment:
now that `task-definition-cron.json` runs the same command as the api task definition, with the addition of `RUN_TASK`, is there a way to just reuse the same json template so we don't have to maintain two?

Reply:
Great idea!